src/classic/clvm_tools/stages/mod.rs

```diff
@@ -5,10 +5,10 @@ pub mod stage_0;
 pub mod stage_1;
 pub mod stage_2;
 
-pub fn run<'a>(allocator: &'a mut Allocator) -> NodePtr {
-    assemble(allocator, &"(a (opt (com 2)) 3)".to_string()).unwrap()
+pub fn run(allocator: &mut Allocator) -> NodePtr {
+    assemble(allocator, "(a (opt (com 2)) 3)").unwrap()
 }
 
-pub fn brun<'a>(allocator: &'a mut Allocator) -> NodePtr {
-    assemble(allocator, &"(a 2 3)".to_string()).unwrap()
+pub fn brun(allocator: &mut Allocator) -> NodePtr {
+    assemble(allocator, "(a 2 3)").unwrap()
```

Comment:
simply that this should be irrefutable.  it's a utility pattern that's hand assembled here as in the original code (which would have thrown if the assemble failed).  I can change it to hand building the data structure in allocator but since allocator can still fail, there may not be a better choice than panic.  I'm uncertain what i'd do if allocator failed while doing it given that the result is expected to exist.